core/src/test/scala/unit/kafka/coordinator/ConsumerCoordinatorResponseTest.scala

```diff
@@ -41,6 +42,8 @@ class ConsumerCoordinatorResponseTest extends JUnitSuite {
   type JoinGroupCallback = (Set[TopicAndPartition], String, Int, Short) => Unit
   type HeartbeatCallbackParams = Short
   type HeartbeatCallback = Short => Unit
+  type CommitOffsetCallbackParams = immutable.Map[TopicAndPartition, Short]
```

Comment:
The default map is the immutable one, so you can replace all the "immutable.Map"s you added with just a Map.